mifosng-android/src/main/res/layout/dialog_fragment_charge.xml

```diff
@@ -73,7 +73,8 @@
                     android:layout_width="0dp"
                     android:layout_height="wrap_content"
                     android:layout_weight="1"
-                    android:background="@color/light_grey" />
+                    android:background="@color/light_grey"
```

Comment:
I have an 💡 
Why not abstract it out to a style and apply to multiple places instead of having ```background``` and ```maxLines``` tag repeat itself over and over ?